components/dot_menu/index.ts

```diff
@@ -138,11 +140,12 @@ type Actions = {
     doAppCall: DoAppCall;
     postEphemeralCallResponseForPost: PostEphemeralCallResponseForPost;
     setThreadFollow: (userId: string, teamId: string, threadId: string, newState: boolean) => void;
+    openAppsModal: (form: AppForm, call: AppCallRequest) => void;
 }
 
 function mapDispatchToProps(dispatch: Dispatch<GenericAction>) {
     return {
-        actions: bindActionCreators<ActionCreatorsMapObject<ActionFunc>, Actions>({
+        actions: bindActionCreators<ActionCreatorsMapObject<any>, Actions>({
```

Comment:
why did the concrete type get replace with `any`?

Reply:
I did it to simplify the typing, since I feel it does not add much value. Nevertheless, I could have changed it to ActionFunc|Action. Do you prefer it that way?